commands/token_e2e_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestTokenGenerateForUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	tokenDescription := model.NewRandomString(10)
+
+	previousVal := s.th.App.Config().ServiceSettings.EnableUserAccessTokens
+	s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableUserAccessTokens = true })
+	defer s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableUserAccessTokens = *previousVal })
```

Comment:
nit: there is no need to save and restore the value of the config as it is initialised on each `s.SetupTestHelper().InitBasic()`